freezer/freezer_url.go

```diff
@@ -42,7 +42,7 @@ func newFreezerSink(u *url.URL) (substrate.AsyncMessageSink, error) {
 		if sse == "aes256" {
 			enc = straw.S3ServerSideEncoding(straw.ServerSideEncryptionTypeAES256)
 		}
-		if sse != "" {
+		if sse != "" && sse != "aes256" {
```

Comment:
Maybe it would be easier to make this a switch statement like this
```go
	switch q.Get("sse") {
	case "aes256":
		enc = straw.S3ServerSideEncoding(straw.ServerSideEncryptionTypeAES256)
	default:
		return nil, fmt.Errorf("unsupported value: %s passed for sse parameter", sse)	
	}
```